### PR TITLE
Add a test of an Android platform view that draws a gradient

### LINKED ISSUE #152376: At least 1 on-device UI test of an Android platform view working on CI
Potentially `webview_flutter`, but if that's too heavyweight I'll hand-write a `tiny_webview_flutter` for the integration test.

### PR DESCRIPTION
**Pull request by matanlurey**
Closes https://github.com/flutter/flutter/issues/152376.

The gradient, for posterity, looks like this:

Let's see if CI agrees!

/cc @johnmccutchan

### CODE REVIEW

File: dev/integration_tests/android_driver_test/android/app/src/main/kotlin/com/example/android_driver_test/GradientPlatformView.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package com.example.android_driver_test
```
Comment (jonahwilliams) [inline dev/integration_tests/android_driver_test/android/app/src/main/kotlin/com/example/android_driver_test/GradientPlatformView.kt:1]: License header?

**Review by jonahwilliams — APPROVED** [review]
LGTM with nit

**Comment by johnmccutchan** [comment]
Woo hoo 🎉

File: dev/integration_tests/android_driver_test/android/app/src/main/kotlin/com/example/android_driver_test/GradientPlatformView.kt (hunk 1 above)
Comment (matanlurey) [reply dev/integration_tests/android_driver_test/android/app/src/main/kotlin/com/example/android_driver_test/GradientPlatformView.kt:7]: Done!

**Pull request merged**